silverback/application.py

```diff
@@ -15,9 +15,25 @@
 
 
 class SilverbackApp(ManagerAccessMixin):
+    """
+    The application singleton. Must be initialized prior to use.
+
+    Usages example::
+
+        from silverback import SilverbackApp
+
+        app = SilverbackApp()
+
+        ...  # Connection has been initialized, can call broker methods e.g. `app.on_(...)`
```

Comment:
did you build these and make sure this part looks ok?
you may need the double tick marks on `app.on()` unless you want it to be cursive.

Reply:
Yes, this renders correctly as just normal backticks inside a code usage example block